[BUG] @inngest/eslint-plugin missing dist

**Describe the bug**
`@inngest/eslint-plugin@0.0.2` is  missing dist folder

![Screenshot 2023-12-28 at 12 51 54 PM](https://github.com/inngest/inngest-js/assets/14151868/18c3db6c-e42a-42ec-90d9-93f26f55e3ab)


**To Reproduce**
Steps to reproduce the behavior:

1. `npm install -D @inngest/eslint-plugin@latest`
2. `eslint ./`
3.  `Failed to load plugin '@inngest' declared in '.eslintrc.js': Cannot find module './node_modules/@inngest/eslint-plugin/dist/index.js'. Please verify that the package.json has a valid "main" entry`

--- a/.changeset/shaggy-elephants-act.md
+++ b/.changeset/shaggy-elephants-act.md
@@ -1,0 +1,5 @@
+---
+"@inngest/eslint-plugin": patch
+---
+
+Fix packages sometimes not shipping dist files if released with multiple packages

--- a/package.json
+++ b/package.json
@@ -1,7 +1,8 @@
 {
   "scripts": {
     "preinstall": "npx only-allow pnpm",
-    "release:all": "pnpm run release:publish && pnpm run release:tag",
+    "release:all": "pnpm run release:build && pnpm run release:publish && pnpm run release:tag",
+    "build": "pnpm run --if-present --recursive build",
     "release:publish": "pnpm run --if-present --recursive release",
     "release:tag": "node scripts/release/tag.js",
     "log": "pnpm run --if-present --recursive log"

